docs/jobs.rst

```diff
@@ -219,16 +219,24 @@ Optional Fields
     have been emitted by upstream actions. Unlike with ``requires`` attribute,
     dependent actions don't have to belong to the same job. ``triggered_by``
     template may contain any pattern allowed in ``command`` attribute.
+    See :ref:`shortdate` for an explantion of shortdate
```

Comment:
Can you put the explanation right below the `shortdate` example here?

Reply:
I'm a bit hesitant because shortdate is not only used by triggered_by but also other fields. I can move the link below the example though.